store/src/main/java/com/dropbox/android/external/store4/StoreBuilder.kt

```diff
@@ -54,6 +54,8 @@ interface StoreBuilder<Key : Any, Output : Any> {
      */
     fun disableCache(): StoreBuilder<Key, Output>
 
+    fun<UnwrappedOutput: Any> fetcherTransformer(fetcherTransformer: (Output) -> FetcherResult<UnwrappedOutput>)
```

Comment:
Would store response work here? Or is it too generic?

Reply:
Store response has an origin, which I don't want to let users override.